### PR TITLE
add update guide for 1.1.0

### PR DESCRIPTION
**Pull request by menski**

### CODE REVIEW

File: docs/guides/update-guide/100-to-110.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
+---
+id: 100-to-110
+title: Update 1.0 to 1.1
+---
+
+The following sections will explain for each component of the system which
+adjustments have to be made to migrate from Camunda Cloud 1.0.x to 1.1.0
```
Comment (akeller) [inline docs/guides/update-guide/100-to-110.md:7]:
```suggestion
adjustments have to be made to migrate from Camunda Cloud 1.0.x to 1.1.0.
```

File: docs/guides/update-guide/100-to-110.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+
+## Server
+
+### Operate
+
+With Operate 1.1 a new feature was introduced to navigate between call activity
+hierarchies. This feature is only available for instances started after the
+version update. Older instances will not expose this information to the user.
```
Comment (akeller) [inline docs/guides/update-guide/100-to-110.md:16]: I always specify the exact version so no one has to read very far for the right context 😄 

```suggestion
version 1.1 update. Older instances will not expose this information to the user.
```

File: docs/guides/update-guide/100-to-110.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+## Client
+
+### Zeebe Java Client
+
+To prepare to support multiple regions in Camunda Cloud SaaS we adopted the
+Zeebe URLs used to connect to your cluster to contain a region sub-domain, i.e
+`${CLUSTE_ID}.zeebe.camunda.io` is now `${CLUSTER_ID}.bru-2.zeebe.camunda.io`.
+
+We are confident that we rolled out this change transparent and as backwards
```
Comment (akeller) [inline docs/guides/update-guide/100-to-110.md:26]:
```suggestion
We are confident that we rolled out this change transparently and as backwards
```

File: docs/guides/update-guide/100-to-110.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+compatible as possible. Still there exists situation which this might impact
+you on your update path. For existing clusters the old URLs still be
+functional, we still recommend to update the configuration of your clients to
+the new URL format.
```
Comment (akeller) [inline docs/guides/update-guide/100-to-110.md:30]:
```suggestion
you on your update path. For existing clusters the old URLs will still be
functional, we recommend you update the configuration of your clients to
the new URL format.
```

**Review by akeller — COMMENTED** [review]
I recommend a couple of small changes, otherwise, ship it! 🚀

**Comment by menski** [comment]
Thanks for the fast review :rocket:

**Pull request merged**